scenario/features/blockevents.feature

```diff
@@ -77,3 +77,24 @@ Feature: Block event listening
             | block                  |
             | filtered block         |
             | block and private data |
+
+    Scenario Outline: Checkpoint of block events
+        Given I create a checkpointer
+        And I use my checkpointer to listen for <type> events
+        And I prepare to submit an echo transaction
+        And I set the transaction arguments to ["checkpoint"]
+        And I invoke the transaction
+        Then I should receive a <type> event
+        Then I should checkpoint the <type> event
```

Comment:
Similar to the chaincode checkpointing PR, we shouldn't have a separate step to do the checkpointing. Since we used a checkpointer to listen for events we should be checkpointing events read from that listener.